selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -703,7 +703,7 @@ public void waitMarkerDisappears(String markerType, int position) {
    * @param markerType is type of the marker
    */
   public void waitAllMarkersDisappear(String markerType) {
-    loaderDriverWait.until(
+    elemDriverWait.until(
```

Comment:
**elemDriverWait** is 3 times less timeout compare to **loaderDriverWait**. So it's hardly ever to help to stabilize the tests.